core/src/main/java/uk/ac/bbsrc/tgac/miso/core/manager/UserAuthMisoRequestManager.java

```diff
@@ -591,9 +587,7 @@ public Study getStudyById(long studyId) throws IOException {
 
   @Override
   public Submission getSubmissionById(long submissionId) throws IOException {
-    Submission o = backingManager.getSubmissionById(submissionId);
-    if (readCheck(o)) return o;
-    else throw new AuthorizationIOException("User " + getCurrentUsername() + " cannot read Submission " + submissionId);
+    return backingManager.getSubmissionById(submissionId);
```

Comment:
The submission will include other entities that the user may not have read permissions on. This means that anything that has been included in a submission becomes readable to all users in a way.

The permissions were not set up correctly before and did not work then either. I'm not sure what the solution is. Make a ticket?